packages/contract-wrappers/test/coordinator_wrapper_test.ts

```diff
@@ -0,0 +1,359 @@
+        exchangeContractAddress = contractWrappers.exchange.address;
+        userAddresses = await web3Wrapper.getAvailableAddressesAsync();
+        zrxTokenAddress = contractWrappers.exchange.zrxTokenAddress;
+        fillScenarios = new FillScenarios(
+            provider,
+            userAddresses,
+            zrxTokenAddress,
+            exchangeContractAddress,
+            contractWrappers.erc20Proxy.address,
+            contractWrappers.erc721Proxy.address,
+        );
+        [, makerAddress, takerAddress, feeRecipientAddress, anotherFeeRecipientAddress] = userAddresses.slice(0, 5);
+        [makerTokenAddress, takerTokenAddress] = tokenUtils.getDummyERC20TokenAddresses();
+        [makerAssetData, takerAssetData] = [
+            assetDataUtils.encodeERC20AssetData(makerTokenAddress),
+            assetDataUtils.encodeERC20AssetData(takerTokenAddress),
+        ];
+
+        // set up mock coordinator server
+        const coordinatorServerConfigs = {
+            HTTP_PORT: 3000,
+            NETWORK_ID_TO_SETTINGS: {
+                50: {
+                    FEE_RECIPIENTS: [
+                        {
+                            ADDRESS: feeRecipientAddress,
+                            PRIVATE_KEY: constants.TESTRPC_PRIVATE_KEYS[
+                                userAddresses.indexOf(feeRecipientAddress)
+                            ].toString('hex'),
+                        },
+                        {
+                            ADDRESS: anotherFeeRecipientAddress,
+                            PRIVATE_KEY: constants.TESTRPC_PRIVATE_KEYS[
+                                userAddresses.indexOf(anotherFeeRecipientAddress)
+                            ].toString('hex'),
+                        },
+                    ],
+                    // Ethereum RPC url, only used in the default instantiation in server.js of 0x-coordinator-server
+                    // Not used here when instantiating with the imported app
+                    RPC_URL: 'http://ignore',
+                },
+            },
+            // Optional selective delay on fill requests
+            SELECTIVE_DELAY_MS: 0,
+            EXPIRATION_DURATION_SECONDS: 60, // 1 minute
+        };
+        app = await getAppAsync(
+            {
+                [config.networkId]: provider,
+            },
+            coordinatorServerConfigs,
+        );
+
+        await app.listen(coordinatorServerConfigs.HTTP_PORT, () => {
+            // tslint:disable-line:await-promise
+            console.log(`Coordinator SERVER API (HTTP) listening on port ${coordinatorServerConfigs.HTTP_PORT}!`); // tslint:disable-line:no-console
+        });
+
+        // setup coordinator registry
+        coordinatorRegistryInstance = new CoordinatorRegistryContract(
+            CoordinatorRegistry.compilerOutput.abi,
+            contractAddresses.coordinatorRegistry,
+            provider,
+            { gas: 6000000 },
```

Comment:
Is this really needed?

Reply:
oops, that was leftover from when I was using the ganache snapshot in docker  instead of in-process ganache. removing.